frontend/src/pages/about.js

```diff
@@ -0,0 +1,160 @@
+        <SmallPrint>
+          (Das sind meine persönlichen Empfehlungen, keine Werbung, keine
+          Bezahlung, kein Affiliate, kein nichts. Bilde dir deine eigene
+          Meinung.)
+        </SmallPrint>
+        <h3>Bücher</h3>
+        <ul>
+          <li>
+            Über Schiffe und U-Boote im Gemeinde-Kontext: <br />
+            <a
+              href="https://www.amazon.de/Die-leisen-Weltver%C3%A4nderer-introvertierter-Christen/dp/3775158286/ref=sr_1_1?__mk_de_DE=%C3%85M%C3%85%C5%BD%C3%95%C3%91&dchild=1&keywords=debora+sommer&qid=1589447299&sr=8-1"
+              target="_blank"
+              rel="noopener noreferrer"
+            >
+              Debora Sommer :: Die leisen Weltveränderer
+            </a>
+          </li>
+          <li>
+            Der Klassiker: <br />
+            <a
+              href="https://www.amazon.de/Still-Kraft-Introvertierten-Susan-Cain/dp/3442157641/ref=sr_1_1?__mk_de_DE=%C3%85M%C3%85%C5%BD%C3%95%C3%91&crid=AYHUIQA1AV8L&dchild=1&keywords=still+susan+cain&qid=1589447457&sprefix=still+susain+%2Caps%2C196&sr=8-1"
+              target="_blank"
+              rel="noopener noreferrer"
+            >
+              Susain Cain :: Still
+            </a>
+          </li>
+          <li>
+            Um auch als leiser Mensch mehr Präsenz zu finden:
+            <br />
+            <a
+              href="https://www.amazon.de/Leise-Menschen-starke-Wirkung-Pr%C3%A4senz/dp/3492306993/ref=sr_1_3?__mk_de_DE=%C3%85M%C3%85%C5%BD%C3%95%C3%91&crid=KIVZK510MM56&dchild=1&keywords=l%C3%B6hken+sylvia&qid=1589447502&sprefix=l%C3%B6hke%2Caps%2C168&sr=8-3"
+              target="_blank"
+              rel="noopener noreferrer"
+            >
+              Susan Löhken :: Leise Menschen - starke Wirkung
+            </a>
+          </li>
+        </ul>
+        <h3>Podcasts</h3>
+        <ul>
+          <li>
+            Vanilla Mind <br />
+            <a
+              href="https://vanilla-mind.de/podcast/"
+              target="_blank"
+              rel="noopener noreferrer"
+            >
+              Still & Stark
+            </a>
+          </li>
+          <li>
+            Die Lösung – der Psychologie-Podcast von PULS <br />
+            <a
+              href="https://www.br.de/mediathek/podcast/die-loesung-der-psychologie-podcast-von-puls/introvertiert/1663322"
+              target="_blank"
+              rel="noopener noreferrer"
+            >
+              Introvertiert (einzelne Folge)
+            </a>
+          </li>
+        </ul>
+      </main>
+    </>
+  )
+}
+
+const IntroStyled = styled.p`
+  font-size: 18px;
+  font-weight: 500;
+  margin-bottom: 24px;
+`
+const SmallPrint = styled.p`
+  font-size: 14px;
+  font-style: italic;
+`
+const ListStyled = styled.ul`
+  margin-bottom: 12px;
+`
+const ListItemStyled = styled.li`
+  margin-top: 0;
+  margin-left: 12px;
+  list-style: none;
+  font-size: 16px;
+`
+const OverviewStyled = styled.h2`
+  margin-top: 12px;
```

Comment:
das könntest du auch unter margin: 12px 0; zusammenfassen

Reply:
nicht ganz, weil ich rechts und unten nicht zu meiner GlobalStyles verändern möchte. Hab's aber jetzt in eine Reihe geschrieben :)